mocks/service/launchpad/index.js

```diff
@@ -44,10 +44,7 @@ router.get('/devel/*snaps', (req, res) => {
 });
 
 router.post('/~:owner/+snap/:name', (req, res) => {
```

Comment:
Ah, double backslash... I didn't dig that deep into the docs.
Thanks for fixing it!